src/sympc/api.py

```diff
@@ -227,6 +230,14 @@
         "sympc.tensor.replicatedshare_tensor.ReplicatedSharedTensor.numel",
         "syft.lib.python.Int",  # FIXME: Can't we just return an int??
     ),
+    (
+        "sympc.tensor.replicatedshare_tensor.ReplicatedSharedTensor.get_ring_size",
```

Comment:
Q: Where is this implemented? (I don't see it in the RST)

Reply:
Its here george:
https://github.com/OpenMined/SyMPC/blob/45274eeb98f297dac04067226b2b801a1072a58a/src/sympc/tensor/replicatedshare_tensor.py#L146